mautic/mautic
# [UX] Improve the "Use owner as mailer?" tooltip to tell users where to find the option

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This is a small change in the tooltip text to tell the user where they can find the field that defines who owns the contact, while also improving clarity.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Emails
3. Create new or edit
4. Go to the Advanced tab
5. Look at the Use contact owner as mailer label and tooltip

![image](https://github.com/mautic/mautic/assets/116097999/fa054374-034a-4263-8e38-00db4755a14e)
